src/components/CommentFooter/CommentFooter.less

```diff
@@ -16,6 +16,21 @@
     margin: 0 8px;
   }
 
+  &__bullet_br:after {
+    content: "\2022";
+    font-size: 14px;
+    margin: 0 8px;
+	  display: block;
+	  visibility: hidden;
+  	line-height: 0.5;
+	
+	  @media @very-small {
```

Comment:
This line is breaking the deployment of the app with this issue:
```
oader/lib/index.js??postcss!node_modules/less-loader/dist/cjs.js!src/components/CommentFooter/CommentFooter.less:
       1 module
       
       ERROR in ./node_modules/css-loader?{"importLoaders":1}!./node_modules/postcss-loader/lib?{"ident":"postcss","plugins":[null]}!./node_modules/less-loader/dist/cjs.js!./src/components/CommentFooter/CommentFooter.less
       Module build failed:
       
       
       @media @very-small {
       ^
       Variable @very-small is undefined
       in /tmp/build_f410f0737e336c27aa8dc7e1c220ce61/busyorg-busy-bb35cb9/src/components/CommentFooter/CommentFooter.less (line 27, column 10)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! busy@2.0.0 build:client: `webpack -p --config ./webpack/webpack.prod.config.js`
npm ERR! Exit status 2
```

Reply:
the variable file is not imported i will fix it .